index.html

```diff
@@ -2794,6 +2794,13 @@ <h3 id="lifetime-AudioNode" class="informative">
             <em>playing</em> reference to themselves while they are currently
             playing.
             </li>
+            <li>An <dfn>active reference</dfn> for
+            <a><code>AudioWorkletNode</code></a>s. This node maintains an <em>
+              active</em> reference to itself upon creation. This reference is
+              subsequently maintained or removed based on the return values
+              from invocation of its associated <a>AudioWorkletProcessor</a>'s
+              <code>process()</code> method.
```

Comment:
The `active reference` here means the JS reference?

If so, I am not sure this paragraph is correct. Even if the `process()` method returns `false`, the instance should not get collected if the JS reference to the node is dropped.

> `the reference is ... removed based on the return values from invocation ...`

This reads like the `process()` method returns false value can remove the JS reference from the global scope. From my perspective, the instance of AWN and AWP are ready to be collected when 1) the JS reference to the AWN is dropped and 2) the AWP's process() returns false. The timing of collection is up to the user agent, so we simply mark this instance as 'collect when possible'.